File: algo-contest-1-17-07-2022/B/solution.py
queries = int(input())

while queries:
    queries -= 1
    n = int(input())
    a = list(map(int, input().split()))

    a = sorted(a)
    ans = 0
    for i in range(1, len(a)):
        ans += a[i] - a[0]
    
    print(ans)

File: algo-contest-1-17-07-2022/B/solution.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int queries;
    cin >> queries;

    while (queries--) {
        int n;
        cin >> n;
        vector < int > a(n);
        for (int &item: a)
            cin >> item;

        sort(a.begin(), a.end());

        int val = a[0], ans = 0;
        for (int item: a)
            ans += item - val;

        cout << ans << "\n";
    }
    return 0;
}